.github/workflows/windows-artifacts.yml

```diff
@@ -40,6 +40,14 @@ jobs:
         with:
           name: windows-installer
           path: "./out/windows-amd64/crc-windows-installer.zip"
+      - name: Build Linux binary
```

Comment:
Can you create a follow up issue on this one, about to renaming this action (as now it also includes linux build, it would be great to properly name it).

Also ensure linux build is only done in one place